decidim-core/lib/devise/models/.#decidim_validatable.rb

```diff
@@ -0,0 +1 @@
+josepjaume@Joseps-MacBook-Pro.local.742
```

Comment:
Please, remove this file. Looks like it's something generated by your Spacemacs configuration?